test/cljs/rems/administration/test_create_form.cljs

```diff
@@ -212,7 +223,17 @@
                :field/placeholder {:en "en placeholder"
                                    :fi "fi placeholder"}}]
 
-             (mapv #(build-request-field % languages) fields-empty-info-text))))))
+             (mapv #(build-request-field % languages) fields-empty-info-text)))
+      (is (= [{:field/id "fld1"
+               :field/title {:en "en title"
+                             :fi "fi title"}
+               :field/optional true
+               :field/type :text
+               :field/max-length 12
+               :field/placeholder {:en "en placeholder"
+                                   :fi "fi placeholder"}}]
+
+             (mapv #(build-request-field % languages) fields-one-empty-info-text))))))
```

Comment:
Hmm, I would prefer for `:field/info-text` to be present in this case, which would trigger a validation error for it from the backend, instead of just silently dropping it.

This isn't such a big deal because we also run the validations in the frontend so you should never get this far.

Reply:
Yeah I think not sending it is the right thing for the UI to do, but it does leave the backend untested. Backend could have its own test case for this but maybe not worth it.

Reply:
We have a test for the common form-validation code anyway.

Reply:
Do you think I could do that in a different PR?
I do not think that we currently test any field at this stage?

Reply:
it's fine like this, as I said it's not a big deal, we're just thinking out loud with @Macroz

Reply:
what do you think @Macroz ?